code401challenges/src/main/java/code401challenges/tree/Node.java

```diff
@@ -0,0 +1,20 @@
+package code401challenges.tree;
+
+public class Node {
+    int data;
+    Node left;
+    Node right;
+
+
+    public Node(int data, Node left, Node right) {
+        this.data = data;
+        this.left = left;
+        this.right = right;
+    }
+
+    public Node(int data){
+        this.data = data;
+        this.left = null;
+        this.right = null;
+    }
```

Comment:
To string method;
`    @Override
    public String toString() {
        return "Node{" +
                "data=" + data +
                '}';
    }`